deploy/role_binding.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 subjects:
 - kind: ServiceAccount
   name: marketplace-operator
-  namespace: openshift-marketplace
+  namespace: marketplace
```

Comment:
How does this introduce a bug? I recognize that this should be done in a separate PR but I don’t see the bug. Are we supposed to deploy to the openshift-marketplace namespace? I only ask because oc fails when running oc new-project openshift-marketplace.

Reply:
Because you now make it inconsistent with some of the other yamls like the OperatorSource example that QE is using. We need to create `openshift-marketplace` using the installer for 4.0. In the interim, we can use `kubectl` to create that namespace.